### DISCUSSION TITLE
best configuration for speaker diarization

### DISCUSSION THREAD

**Discussion by muizzhaque** [NVIDIA-NeMo/NeMo#7100] [Q&A]
I have been using speaker diarization funtion from config file present in the tutorial and i want to get better results than what is given by default. What would be the better ombination for the following parameters under configuration file:-
     cfg.diarizer.vad.parameters.onset = 0.8
     cfg.diarizer.vad.parameters.offset = 0.6
     cfg.diarizer.vad.parameters.pad_offset = -0.05
     cfg.diarizer.msdd_model.parameters.sigmoid_threshold = [0.7, 1.0]
    cfg.diarizer.speaker_embeddings.parameters.window_length_in_sec = [1.5,1.25,1.0,0.75,0.5] 
    cfg.diarizer.speaker_embeddings.parameters.shift_length_in_sec = [0.75,0.625,0.5,0.375,0.25] 
    cfg.diarizer.speaker_embeddings.parameters.multiscale_weights= [1,1,1,1,1] 
    
    one more to add is that for cfg.diarizer.asr.parameters.asr_based_vad = False will it help if i use True

**Comment by jeremy110** [comment]
If you simply want to run speaker diarization, then the parameters of automatic speech recognition (ASR) are completely unrelated.

If you find that the results from voice activity detection (VAD) are not satisfactory, you can adjust the onset and offset parameters.
